TradeItIosTicketSDK2/TradeItPortfolioEquityPositionsTableViewCell.swift

```diff
@@ -13,10 +13,16 @@ class TradeItPortfolioEquityPositionsTableViewCell: UITableViewCell {
     @IBOutlet weak var dayLabelValue: UILabel!
     @IBOutlet weak var totalLabelValue: UILabel!
     @IBOutlet weak var totalReturnLabelValue: UILabel!
+    @IBOutlet weak var buyButton: UIButton!
+    @IBOutlet weak var sellButton: UIButton!
     
     
+    weak var delegate: TradeItPortfolioPositionsTableViewCellDelegate?
+    
+    private var selectedPosition: TradeItPortfolioPosition!
     
     func populate(withPosition position: TradeItPortfolioPosition) {
+        self.selectedPosition = position
```

Comment:
Make sure that trade buttons (and swipe-to-trade) only show up for equity positions.  There is now an `instrumentType` extension on `TradeItPosition`/`TradeItFxPosition`.